### PR TITLE
Show RBAC error on preflight page

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: pkg/apiserver/server.go [hunk 1]
```diff
@@ -56,6 +56,7 @@ func Start() {
 	r.Path("/api/v1/app/airgap").Methods("OPTIONS", "POST", "PUT").HandlerFunc(handlers.UploadAirgapBundle)
 
 	// Implemented handlers
+	r.Path("/api/v1/preflight/{appSlug}/{clusterSlug}/{sequence}/ignore-rbac").Methods("OPTIONS", "POST").HandlerFunc(handlers.IgnorePreflightRBACErrors)
```
Comment (divolgin) [inline pkg/apiserver/server.go]: I'm open to suggestions here for API routes.

File: pkg/downstream/downstream.go [hunk 2]
```diff
@@ -8,13 +8,32 @@ import (
...
+}
+
 func ListDownstreamsForApp(appID string) ([]*types.Downstream, error) {
 	db := persistence.MustGetPGSession()
 	query := `select cluster_id, downstream_name, current_sequence from app_downstream where app_id = $1`
 	rows, err := db.Query(query, appID)
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to get downstreams")
 	}
+	defer rows.Close()
```
Comment (divolgin) [inline pkg/downstream/downstream.go]: This is an unrelated DB connection leak

File: pkg/apiserver/server.go (hunk 1 above)
Comment (laverya) [reply pkg/apiserver/server.go]: if we're going to have any other preflight routes, we should add an extra constant path here
unless this router will route `/api/v1/preflight/appslug/clusterslug/sequence/other-preflight` properly?

File: pkg/downstream/downstream.go [hunk 3]
```diff
@@ -58,3 +78,36 @@ func SetDownstreamVersionPendingPreflight(appID string, sequence int64) error {
...
+	}
+
+	if !shouldIgnore.Valid {
+		return false, nil
+	}
+
+	return shouldIgnore.Bool, nil
+}
+
+func SetIgnorePreflightPermissionErrors(appID string, clusterID string, sequence int64) error {
```
Comment (laverya) [inline pkg/downstream/downstream.go]: should this share the name of the above function? Like, have one be 'set' and the other 'get' or something
